15_1_Building_the_Regtest.md

```diff
@@ -0,0 +1,31 @@
+# 15.1: Building the Regtest
```

Comment:
I'm not sure what this title is insinuating. You don't build regtest, and the use of `the` is weird.